src/MessagingAttachment/MessagingAttachment.tsx

```diff
@@ -33,6 +40,9 @@ export const MessagingAttachment: React.FC<Props> = ({
   isUpload,
   uploadComplete,
 }: Props) => {
+  const [attachmentPreviewIsShowing, setAttachmentPreviewIsShowing] = React.useState(false);
+  const isImageAttachment = fileTypeToIcon(fileType) === "image";
```

Comment:
How come this is using a function called `fileTypeToIcon` but it returns a string instead of an icon?

Reply:
yeah that name is misleading. it returns the type of icon to use, the name of the icon if you will. i should change the name of the function to `fileTypeToIconName` or something?

Reply:
Yeah maybe `fileTypeToIconType` as it seems `image` would be a `type`? Im not too sure what the other response types are though so whatever you think is best is good and seems like an improvement over just `Icon`

Reply:
that's what i ended up doing 👍